frontend/packages/dev-console/src/components/add/AddPageLayout.scss

```diff
@@ -0,0 +1,20 @@
+.odc-add-page-layout-hint-block {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    white-space: nowrap;
+    &__text {
+      margin-right: var(--pf-global--spacer--sm);
+    }
+    &__details-switch {
+      display: flex;
+      margin-left: auto;
+      align-items: center;
+      &__text {
+        display: inline;
+        vertical-align: middle;
+        margin-left: var(--pf-global--spacer--xs);
+      }
+    }
+  }
+  
```

Comment:
nit: missing new line at end of file

Reply:
There was a trailing space in the new line at the end of the file.
fixed it.